src/main/java/data/Account.java

```diff
@@ -40,6 +44,19 @@ public Account(String accountID, String vendorID, String entity,
     this.aggregationMode = aggregationMode;
   }
 
+  public Account(HttpServletRequest request, String accountID, String vendorID) {
```

Comment:
Could accountID and vendorID potentially be read off of the request object?

Reply:
that is sort of the plan, but that is not currently working on the front-end. I can make sure we get that resolved in the next PR